### PR TITLE
Update TxFilterStub.cpp

### PR DESCRIPTION
**Pull request by Gillou68310**

### CODE REVIEW

File: src/GraphicsDrawer.h [hunk 1]
```diff
@@ -98,7 +98,7 @@ class GraphicsDrawer
 		u32 dstHeight;
 		bool invertX = false;
 		bool invertY = false;
-		std::array<CachedTexture *, 2> tex{ { nullptr, nullptr } };
```
Comment (gonetz) [inline src/GraphicsDrawer.h]: Why? I use VS2013 and have no problems with this code.

Comment (Gillou68310) [reply src/GraphicsDrawer.h]: Weird! Here's the error I get:
```
error C2797: 'GraphicsDrawer::CopyRectParams::tex': list initialization inside member initializer list or non-static data member initializer is not implemented
```

What is your compiler version?

Here's mine:
image

Comment (gonetz) [reply src/GraphicsDrawer.h]: C:\Program Files (x86)\Microsoft Visual Studio 12.0>cl
Оптимизирующий компилятор Microsoft (R) C/C++ версии 18.00.21005.1 для x86

Seems to be mine is older than yours: 18.00.21005.1

Comment (gonetz) [reply src/GraphicsDrawer.h]: I have similar error when I'm trying to init local variables with initialization list, but this part works ok.

Comment (Gillou68310) [reply src/GraphicsDrawer.h]: https://msdn.microsoft.com/en-us/library/dn793970.aspx

Comment (gonetz) [reply src/GraphicsDrawer.h]: Ok, how about:

	struct CopyRectParams
	{
		s32 srcX0 = 0;
		s32 srcY0 = 0;
		s32 srcX1;
		s32 srcY1;
		u32 srcWidth;
		u32 srcHeight;
		s32 dstX0 = 0;
		s32 dstY0 = 0;
		s32 dstX1;
		s32 dstY1;
		u32 dstWidth;
		u32 dstHeight;
		bool invertX = false;
		bool invertY = false;
		typedef std::array<CachedTexture *, 2> Textures;
		Textures tex = Textures{ { nullptr, nullptr } };
		graphics::CombinerProgram * combiner = nullptr;
		graphics::TextureParam filter;
	};

It works with my version of compiler.

Comment (Gillou68310) [reply src/GraphicsDrawer.h]: It's working for me too

Comment (Gillou68310) [reply src/GraphicsDrawer.h]: I'll update my PR

Comment (gonetz) [reply src/GraphicsDrawer.h]: Ok. If this works, other changes from "Fix VS2013 compilation" commit are useless.

Comment (Gillou68310) [reply src/GraphicsDrawer.h]: Sure

Comment (Gillou68310) [reply src/GraphicsDrawer.h]: Done

**Pull request merged**